src/main/kotlin/controllers/PersonController.kt

```diff
@@ -23,6 +22,7 @@ import org.springframework.web.bind.annotation.RequestBody
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 import org.springframework.web.server.ResponseStatusException
+import java.security.Principal
 
```

Comment:
The import order matters here. The previous implementation was the correct lexicographic order
   [ktlint] /Users/tkgroot/projects/flock/flock-eco-holidays/src/main/kotlin/controllers/PersonController.kt
   [ktlint]   3:1 Imports must be ordered in lexicographic order without any empty lines in-between
   [ktlint] "checkstyle" report written to /Users/tkgroot/projects/flock/flock-eco-holidays/ktlint.xml